docker/Makefile

```diff
@@ -9,4 +9,4 @@ build:
 
 .PHONY: test-invokation
 test-invokation: build
-	docker run --rm $(IMAGE_NAME)
+	docker run --rm -i $(IMAGE_NAME) --debug
```

Comment:
This has nothing to do with this commit. Pls fixup the previous one.

Reply:
for reviewers: this is fixed.